server/routes/index.js

```diff
@@ -56,6 +57,7 @@ r.get('/series/(W):id', renderSeries);
 r.get('/webcomic-series/:id', renderWebcomicSeries);
 r.get('/info/opening-times', renderOpeningTimes);
 r.get('/pages/:id', renderPage);
+r.get('/info/newsletter', renderNewsletterPage);
```

Comment:
I think we might want to chat about `/info` pages, as they don't exist any more.